src/index.ts

```diff
@@ -79,10 +80,14 @@ class UP9Monitor {
     }
 
     requestLogger = (httpModule, protocol) => {
-        let original = httpModule.request
-        httpModule.request = (request, callback) => {
+        httpModule.request = this.getHttpTappedFunc(httpModule.request, protocol);
```

Comment:
does it mean we only capture requests when use do the "get" or the "request" that used? (I guess there is also "put" / "post"..?)

Reply:
.get is a quick method for GET requests, there are no .post/.patch etc
.request is used for all other methods

Reply:
weird that they do not have it